app/controllers/users_controller.rb

```diff
@@ -75,6 +75,39 @@ def destroy
 
   private
 
+    def user_params
+      if current_user && current_user.admin?
+        params.require(:user).permit!
```

Comment:
True, I could whitelist the specific admin tier variables, but at this point an admin can modify any user param from the web interface anyway, so the results would be same.